manual/manual_03.asciidoc

```diff
@@ -1709,11 +1712,16 @@ Container
 --------------------------------------------------------------------------------
 
 
+// @FIXME: The following note is not very clear, what does it mean by "things
+//         which are not visible"? invisible to who or what? to the player? or
+//         to the LOCATE LIMITS? And, how would that differ/affect LIMITS in
+//         respect to listed instances from inside the container?
+//         Please,clarify! adding an example if needed.
 
 [NOTE]
 ================================================================================
 The `Count` limit considers all instances in the container.
-This might differ from the number of instances listed e.g. if the container takes ``Thing``s (which are not 'visible').
+This might differ from the number of instances listed e.g. if the container takes ``Thing``s (which are not '`visible`').
 ================================================================================
```

Comment:
@thoni56, this note is not very clear. I've added some comments above the admonition/NOTE to illustrate the unclear points.